Makefile

```diff
@@ -9,6 +9,7 @@ clean:
 .PHONY: lint
 lint:
 	flake8 evalml && isort --check-only evalml && python docs/notebook_version_standardizer.py check-versions
+	pydocstyle evalml/ --convention=google --add-ignore=D107 --add-select=D400 --match-dir='^(?!(tests)).*'
```

Comment:
Reads: use Google convention (since that was mostly what our convention was) but add D400 (add period to first line). Match all directories that don't start with `test`.